slides/developer-guide/working-with-documents/document-conversion/using-html5-templates/convert-using-custom-html5-templates/_index.md

```diff
@@ -0,0 +1,348 @@
+---
+title: "Convert Using Custom HTML5 Templates"
+keywords:
+- PowerPoint
+- presentation
+- REST API
+- cloud API
+- convert a presentation
+- export a presentation
+type: docs
+url: /convert-using-custom-html5-templates/
+weight: 10
+---
+## **Introduction**
+
+Presentations are converted to HTML5 using html (Razor) templates. You can upload and use your own templates to override default conversion layout and/or styles.
+
+To use custom templates, you need to,
+
+1. Create a storage folder.
+1. Upload your custom HTML5 templates to that folder.
+1. Call one of the [conversion](/slides/convert-presentations/) methods and specify the storage folder as **TemplatesPath** option value.
+
+Only the last step is repeated for every conversion. The rest are done once; when you have uploaded your templates, you use them as long as you need. You have to upload them again only when your templates are modified and need to be refreshed.
+
+You need not have the whole set of templates in your templates folder. E.g. if you wish to render picture frames in a customized way, and leave all the rest as they are, you only need to provide your custom *pictureFrame.html* file. The application will use default files for all the other templates.
+
+Instead of creating the template(s) from scratch, you can get the default template set by using [GetHtml5Templates](/slides/get-default-html5-templates/) method, and then modify the template files you need.
+
+The example below shows conversion to HTML5 using custom PictureFrame template.
```

Comment:
I suggest to replace
"The example below shows conversion to HTML5 using custom PictureFrame template."
with
"The example below shows the conversion to HTML5 using a custom PictureFrame template."